python/ray/dataframe/dataframe.py

```diff
@@ -1676,4 +1676,6 @@ def to_pandas(df):
     Returns:
         A new pandas DataFrame.
     """
-    return pd.concat(ray.get(df._df))
+    coned_df = pd.concat(ray.get(df._df))
```

Comment:
What is `coned`?

Reply:
a scheme/lisp era jargon: for concatenate. I changed to joined.